Plex/grab-all-posters.py

```diff
@@ -182,7 +182,8 @@ def get_posters(item, artwork_path, tmid, tvid):
 
                     poster_obj = {}
                     tgt_ext = ".dat" if ID_FILES else ".jpg"
-                    tgt_file_path = f"{tmid}-{tvid}-{item.ratingKey}-{get_SE_str(item)}-{str(idx).zfill(3)}{tgt_ext}"
+                    # tgt_file_path = f"{tmid}-{tvid}-{item.ratingKey}-{get_SE_str(item)}-{str(idx).zfill(3)}{tgt_ext}"
+                    tgt_file_path = f"{get_SE_str(item)}-{tmid}-{tvid}-{item.ratingKey}-{str(idx).zfill(3)}{tgt_ext}"
```

Comment:
Won't this end up with anything that isn't an episode getting named with a leading `-`?

Reply:
Indeed... just saw that

Reply:
I think the right answer here might be a "USE_ASSET_NAMING" toggle to build out the asset-folder-style hierarchy.

Reply:
So you are suggesting a toggle in the .env? Might be extra options for no additional value...since most people will want to read the folders with alpha and not the tmdbid as the prefix. Unless I have misunderstood what the toggle you are proposing is referring to.

Reply:
I just proposed a small function to remove the leading dash. Sent in the new commit

Reply:
There are two changes in here; one changes the folder naming to put them in alpha order by title [solves the "find this show" issue and makes sense], and then the other is changing the individual file names to make the S01E01 the primary sort, which it basically already is since everything else in that filename is constant.

That latter change makes it seem like you're aiming at a naming convention that's intended for direct use by the asset pipeline; I'm not sure I understand the rationale there otherwise.

My initial goal was to get a directory full of images that one could then choose from and move/rename as appropriate to the individual's use case, so the file naming is really intended to display the ID information we have about a thing as economically as possible.

Reply:
What's the advantage of:
```
S01E01-89393-364080-389-001.jpg
```
over
```
89393-364080-389-S01E01-001.jpg
```
in your view?

Reply:
I prefer the first one as it is more standard when you look at episode file naming out there. Purely esthetic. I also think that this:
`S01E01-001.jpg` would be even better, however, I understand that when people are not using a folder structure the first and the one I proposed here would be probelmatic. I however, would ALWAYS use this script with a folder structure.

Reply:
In my view, they both display in the same order, and the episode images are always grouped together in folders, so I don't see the value in changing the naming on everything.  In either case, one is most likely not going to use the file named as-is; it will likely be renamed, so renaming it from `bing-s01e01` or from `s01e01-bing` doesn't seem significant to me.

I'm been trying to _reduce_ the special-casing between naming movie/show/season/episode, not add more, and this just adds another maintenance point for very little value in my view.

That was the impetus behind the `ASSET_NAMING` toggle notion.  If one wants pretty human-readable names aimed at being used as-is for some purpose, do it across the board with a single setting.

Reply:
I would be totally fine having only this change:
`                    # dir_name, msg = validate_filename(f"{tmid}-{item.title}")
                    dir_name, msg = validate_filename(f"{item.title}-{tmid}")
`

Reply:
Yeah, that part makes total sense generally.

Reply:
And that's been committed.

Reply:
Cool